docs/changelog.md

```diff
@@ -5,11 +5,58 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
-    * [improvement] Added support for Python 3.10
+* [0.27.0](https://pypi.org/project/sematic/0.27.0/)
+    * [feature] Added new S3Location and S3Bucket types that render S3 links in the Dashboard, and
+      documented them
+    * [feature] Added a new Image type that render images in the Dashboard, and documented it
+    * [example] Added TorchLightning + Resnet + Ray example
+    * [example] Added Ray AIR example
+    * [example] Added various enhancements to the example Testing Pipeline
     * [improvement] BREAKING CHANGE: `image_layers` field in `sematic_pipeline` bazel macro
       now ONLY gets passed to image layering, and not also to the Sematic binary target.
       if you are using `image_layers` to express dependencies of a pipeline, you will now
       need to duplicate them in the `deps` field.
+    * [improvement] Added support for Python 3.10
+    * [improvement] Added support for Python 3.10-style type hints
+    * [improvement] The Dashboard login flow will now redirect to the requested page instead of to
+      the homepage
+    * [improvement] Sorted the Runs in the Dashboard Run tree panel by creation time
```

Comment:
Nit: this is a bugfix, it was a regression.

Reply:
Done.